flink-runtime/src/main/java/org/apache/flink/runtime/state/OperatorStateHandle.java

```diff
@@ -36,8 +36,9 @@
 	 * The modes that determine how an {@link OperatorStateHandle} is assigned to tasks during restore.
 	 */
 	public enum Mode {
-		SPLIT_DISTRIBUTE, // The operator state partitions in the state handle are split and distributed to one task each.
-		BROADCAST // The operator state partitions are broadcast to all task.
+		SPLIT_DISTRIBUTE,	// The operator state partitions in the state handle are split and distributed to one task each.
+		BROADCAST,			// The operator state partitions are broadcasted to all tasks.
```

Comment:
Maybe naming this mode `BROADCAST` was not ideal in the first place (perhaps `UNION`, to correspond to the API name, would be better). 
Looking at the name / comments alone between `BROADCAST` and `UNIFORM_BROADCAST` is actually quite confusing.